package.json

```diff
@@ -12,13 +12,12 @@
     "url": "https://github.com/feross/webtorrent-hybrid/issues"
   },
   "dependencies": {
-    "create-torrent": "^3.22.0",
-    "webtorrent": "0.x",
-    "webtorrent-cli": "1.x",
-    "wrtc": "0.0.x"
+    "electron-webrtc": "^0.1.0",
+    "webtorrent": "^0.85.1",
```

Comment:
Please keep the webtorrent version set to `0.x` so we pull in minor version updates.

For example, `^0.85.1` will not install `0.86.0` or `0.87.0`. Semver is confusing for versions less than 1.0.0. We'll fix this by releasing webtorrent 1.0.0 very soon.